### PR TITLE
Document anyNamed and captureAnyNamed.

### PR DESCRIPTION
**Pull request by srawlins**

### CODE REVIEW

File: lib/src/mock.dart [hunk 1]
```diff
@@ -656,6 +656,12 @@ class _VerifyCall {
       'VerifyCall<mock: $mock, memberName: ${verifyInvocation.memberName}>';
 }
 
+// An argument matcher that acts like an argument during stubbing or
```
Comment (natebosch) [inline lib/src/mock.dart]: Triple slash for doc comments

Comment (srawlins) [reply lib/src/mock.dart]: This is not a doc comment. This class is package-private, and not exported in the public interface.

Comment (natebosch) [reply lib/src/mock.dart]: Private and package private members can be documented too. The doc comment is still useful within the package in your IDE.

Comment (srawlins) [reply lib/src/mock.dart]: Is there a way to prevent this class (or its doc comment) from being documented on Pub?

Comment (natebosch) [reply lib/src/mock.dart]: The `lib/src` mechanism which makes it package private should prevent it from showing up in the API docs on pub.

Comment (srawlins) [reply lib/src/mock.dart]: Ah, good. Thanks!

**Pull request merged**